### PR TITLE
Add '/b' flag to hide shell window on Win32

### PR DESCRIPTION
**Pull request by ddantzler**
[Per sindresorhus](https://github.com/sindresorhus/opn/pull/63#issuecomment-327836964), this should be default.

### CODE REVIEW

**Comment by kevva** [comment]
Could you add a link to the comment just for documentation sake?

File: index.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ module.exports = (target, opts) => {
 		}
 	} else if (process.platform === 'win32' || isWsl) {
 		cmd = 'cmd' + (isWsl ? '.exe' : '');
-		args.push('/c', 'start', '""');
+		args.push('/c', 'start', '""', '/b');
```
Comment (kevva) [inline index.js]: `args.push('/c', '/b', 'start', '""');`

Comment (ddantzler) [reply index.js]: That’s not proper, because /b is a flag for start, not cmd.

Comment (kevva) [reply index.js]: Oh, then ignore my comment :).

**Comment by kevva** [comment]
Thanks 👍

**Pull request merged**